### DISCUSSION TITLE
Naming Convention for Tags

### DISCUSSION THREAD

**Discussion by mmcky** [executablebooks/meta#173] [General]
I would like to suggest that we adopt some convention for tags. 

I have noticed that `output_scroll` is used to specify a scrollable output block, when most other tags follow a different pattern such as `hide-output` (https://jupyterbook.org/reference/cheatsheet.html#tags)

Seems that most tags loosely follow `<action>-<target>` pattern and use `-` rather than `_`.

I would propose we change (or at least introduce a synonym for `output_scroll` as `scroll-output`)

**Comment by choldgraf** [comment]
yeah I agree - `output_scroll` is a carryover from the old Jupyter Book days, we just haven't paid it much attention. I'm a fan of adding `scroll-output` and switch the docs to that. More generally `<action>-<thing>` seems reasonable to me

**Reply by mmcky** [reply]
Thanks @choldgraf 👍 -- I will look at adding that synonym soon.
